crates/biome_js_analyze/src/lint/nursery/use_sorted_classes/sort.rs

```diff
@@ -126,11 +126,26 @@ fn compare_classes(a: &ClassInfo, b: &ClassInfo) -> Ordering {
 }
 
 /// Sort the given class string according to the given sort config.
-pub fn sort_class_name(class_name: &TokenText, sort_config: &SortConfig) -> String {
+pub fn sort_class_name(
+    class_name: &TokenText,
+    sort_config: &SortConfig,
+    ignore_last: bool,
+) -> String {
     // Obtain classes by splitting the class string by whitespace.
-    let classes = class_name.split_whitespace().collect::<Vec<&str>>();
+    let mut classes = class_name.split_whitespace().collect::<Vec<&str>>();
     let classes_len = classes.len();
 
+    // If the last class should be ignored, store it and remove it from the classes list.
+    let last_class_strs = if ignore_last {
+        classes[classes_len - 1]
+    } else {
+        ""
+    };
```

Comment:
Here are few suggestions to make the code safer and more idiomatic:
- `classes` can be `0`, don't do `classes_len - 1` because it will panic
- don't return an empty string, use `Option` instead

```rs
let last_class_strs = ignore_last
  .then_some(classes.get(classes_len -1));
```

[`.get`](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.get) return `None` if the index is out of bounds

You'll have to change line 180 to

```rs
if let Some(last_class_strs) = last_class_strs {
	sorted_classes.push(last_class_strs);
}
```